src/blocks/common.blocks/content/_view/content_view_showcase.bemtree

```diff
@@ -0,0 +1,10 @@
+block('content').mod('view', 'showcase')(
+    content()(function () {
+        var data = this.data.node.route.conditions;
+
+        return {
+            block: 'iframe',
+            src: '/__example/' + data.lib + '/' + data.version + apply('path')
```

Comment:
why not? You'll add custom `path` mode on your higher level of definition. But you can add an empty `path` mode here with comments